flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/JoinTest.scala

```diff
@@ -55,12 +55,13 @@ class JoinTest extends TableTestBase {
           unaryNode(
             "DataStreamCalc",
             streamTableNode(1),
-            term("select", "a", "b", "proctime")
+            term("select", "a", "b", "-(proctime, 3600000) AS -",
```

Comment:
@fhueske Actually, from an optimizer perspective this behavior is correct. But we don't want to push down predicates with time indicators. The time indicators are not materialized in this case as the materialization happens before optimization.